client/my-sites/domains/domain-management/domain-overview-pane/style.scss

```diff
@@ -12,4 +12,56 @@
 .a4a-layout.domains-overview .domains-overview__details .section-nav-tabs__list {
     box-sizing: border-box;
     overflow-x: auto;
+}
+
+.item-preview__header-actions .button {
+	border-radius: 4px;
+	height: auto;
+}
+
+.a4a-layout.domains-overview .item-preview__header {
+	.item-preview__header-info .item-preview__header-title-summary .item-preview__header-title {
+		font-family: Recoleta, sans-serif;
+		font-weight: 400;
+		line-height: 1;
+		letter-spacing: normal;
+	}
+
+	.item-preview__header-content {
+		.item-preview__header-info {
+			flex-wrap: wrap;
+		}
+
+		@media (max-width: $break-large) {
+			padding: 0;
+		}
+	}
+
+	.item-preview__header-content .item-preview__header-title-summary .item-preview__header-summary {
+		display: flex;
+		flex-wrap: wrap;
+		gap: 0.5rem;
+		.item-preview__header-summary-link {
+			color: var(--studio-gray-70, #3c434a);
+			text-decoration: none;
+			&:hover {
+				color: var(--color-accent, #3858e9);
+			}
+		}
+	}
+
+	.item-preview__header-actions .button {
```

Comment:
We repeat this selector with the style applied [on line 17th](https://github.com/Automattic/wp-calypso/pull/97481/files#diff-8bb4f3f403bd6b260e3987bef17afc8f2318d3c0b702461dcb5c724e0b46adf9R17).